### ISSUE TITLE
Open New Jira Ticket to track PRs created by NuKeeper 

### ISSUE THREAD

**Issue by dcook-net** [NuKeeperDotNet/NuKeeper#515]
NuKeeper is doing a fantastic job of automating our dependency updates and keeping all of our services up-to-date. However, it has created new issue for us in the fact that this work goes unseen by management, as there are no Jira tickets tracking the work this does for us.

It would great to be able to track changes applied by NuKeeper through the various stages of the Jira board (Test, Ready for deployment, deployed etc - will be different for each team, obviously, and NuKeeper doesn't need to know the details) and have the work that NuKeeper does for us included in team metrics.

**Comment by dcook-net** [comment]
@AnthonySteele was planning on starting to work on this this week, but wanted to know what you thought about it first, see if you had any steer etc.

**Comment by AnthonySteele** [comment]
Agreed it is better left to a webhook, or the new GitHub actions system, not integrated into NuKeeper.

**Comment by dcook-net** [comment]
I'll look into this. Thanks @shep1987

**Comment by MarcBruins** [comment]
Can this be closed?
